Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 129,
```

Comment:
Should it be 130 now; or you plan to port this to 129

Reply:
My recommendation: Lets port this to 129

Reply:
I'll cherry pick to 129